content_scripts/mode_find.coffee

```diff
@@ -60,14 +60,15 @@ class FindMode extends Mode
     matchCount: 0
     hasResults: false
 
-  constructor: (@options = {}) ->
+  constructor: (options = {}) ->
+    console.log "constructor", @options
```

Comment:
`console.log`ging in content scripts is distracting/hostile to web devs. I'll remove this.